modules/rostests/apitests/user32/LoadImage.c

```diff
@@ -1,6 +1,18 @@
 
 #include "precomp.h"
 
+static BOOL AreIconsEqual(HICON h1, HICON h2)
+{
+    BOOL result = FALSE;
+    BOOL (WINAPI*pfnSHAreIconsEqual)(HICON, HICON);
+    HMODULE hMod = LoadLibraryA("SHLWAPI");
+    pfnSHAreIconsEqual = GetProcAddress(hMod, (char*)548);
```

Comment:
```suggestion
    pfnSHAreIconsEqual = (void*)GetProcAddress(hMod, MAKEINTRESOURCEA(548));
```
MSVC sneezes if you don't do a cast to "something" there...
(and I just took what was done there: https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rostests/apitests/shlwapi/SHAreIconsEqual.c;hb=59f0ec1fdf4d5bcfbdc17aa0bae5208dd4e62996#l44 for a solution.)